#!/usr/bin/python3

# The try block does not raise any errors, so the else block is executed:

try:
    print("Hello")
except:
    print("Somthing else went wrong")
else:
    print("Nothing went wrong")

